libraries/libfc/include/fc/time.hpp

```diff
@@ -78,17 +78,16 @@ namespace fc {
         constexpr explicit time_point_sec(uint32_t seconds )
         :utc_seconds(seconds){}
 
-        constexpr time_point_sec( const time_point& t )
+        constexpr explicit time_point_sec( const time_point& t )
         :utc_seconds( t.time_since_epoch().count() / 1000000ll ){}
 
         static constexpr time_point_sec maximum() { return time_point_sec(0xffffffff); }
         static constexpr time_point_sec min() { return time_point_sec(0); }
 
-        constexpr operator time_point()const { return time_point( fc::seconds( utc_seconds) ); }
+        constexpr time_point to_time_point()const { return time_point( fc::seconds( utc_seconds) ); }
```

Comment:
Actually I liked the cast operator better personally. This is more explicit, but whenever we switch a member from `time_point_sec` to `time_point` or vice versa it would require changes throughout. I feel that an implicit conversion between these two time point formats may be warranted.
However, this is just an opinion, not requesting that you change anything.

Reply:
In my opinion, having to make those changes should be required. As illustrated in some of these changes, it looks like the wrong type was used simply because the implicit conversion hid the fact that a conversion was happening.

Reply:
Yes, that is certainly a valid point. It would probably be better to use time_point everywhere.

Reply:
It is better to make the conversion explicit to avoid unintentional conversions.